Makefile

```diff
@@ -1,119 +1,123 @@
-.DEFAULT_GOAL := package
-REPO?=quay.io/costoolkit/os2
-TAG?=dev
-IMAGE=${REPO}:${TAG}
+GIT_COMMIT ?= $(shell git rev-parse HEAD)
+GIT_COMMIT_SHORT ?= $(shell git rev-parse --short HEAD)
+GIT_TAG ?= $(shell git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0" )
+TAG ?= ${GIT_TAG}-${GIT_COMMIT_SHORT}
+REPO?=ttl.sh/elemental-ci
+IMAGE=${REPO}:${GIT_TAG}
 ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 SUDO?=sudo
 FRAMEWORK_PACKAGES?=meta/cos-light
 CLOUD_CONFIG_FILE?="iso/config"
-
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-$$(uname -s)-$$(uname -m) > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
-
-.PHONY: ci
-ci: .dapper
-	./.dapper ci
-
-.PHONY: package
-package: .dapper
-	./.dapper package
+# This are the default images already in the dockerfile but we want to be able to override them
+OPERATOR_IMAGE?=quay.io/costoolkit/elemental-operator:v0.3.0
+SYSTEM_AGENT_IMAGE?=rancher/system-agent:v0.2.9
+TOOL_IMAGE?=quay.io/costoolkit/elemental:v0.0.15-f1fabd4
```

Comment:
Same comment as above: why not using the latest versions?

Reply:
For this one, I need to fix the publish job for elemental-cli as it seems that it's not pushing the proper version on tag, only the tag+sha yikes